disperser/batcher/metrics.go

```diff
@@ -85,43 +85,34 @@ func NewMetrics(httpPort string, logger common.Logger) *Metrics {
 	return metrics
 }
 
-func (g *Metrics) UpdateAttestation(signers, nonSigners int) {
-	g.Attestation.WithLabelValues("signers").Set(float64(signers))
-	g.Attestation.WithLabelValues("non_signers").Set(float64(nonSigners))
+func (g *Metrics) UpdateAttestation(operatorCount, nonSignerCount int) {
+	g.Attestation.WithLabelValues("signers").Set(float64(operatorCount - nonSignerCount))
+	g.Attestation.WithLabelValues("non_signers").Set(float64(nonSignerCount))
 }
 
-// UpdateFailedBatchAndBlobs updates failed a batch and number of blob within it, it only
-// counts the number of blob and batches
-func (g *Metrics) UpdateFailedBatchAndBlobs(numBlob int) {
-	g.Blob.WithLabelValues("failed", "number").Add(float64(numBlob))
-	g.Batch.WithLabelValues("failed", "number").Inc()
-}
-
-// UpdateCompletedBatchAndBlobs updates whenever there is a completed batch. it updates both the
-// number for batch and blob, and it updates size of data blob. Moreover, it updates the
-// time it takes to process the entire batch from "getting the blobs" to "marking as finished"
-func (g *Metrics) UpdateCompletedBatchAndBlobs(blobsInBatch []*disperser.BlobMetadata, succeeded []bool) {
-	totalBlobSucceeded := 0
-	totalBlobFailed := 0
-	totalBlobSize := 0
-
-	for ind, metadata := range blobsInBatch {
-		if succeeded[ind] {
-			totalBlobSucceeded += 1
-			totalBlobSize += int(metadata.RequestMetadata.BlobSize)
-		} else {
-			totalBlobFailed += 1
-		}
+// IncrementCompletedBlobCount increments the number and updates size of processed blobs.
+func (g *Metrics) IncrementCompletedBlobCount(size int, status disperser.BlobStatus) {
```

Comment:
nit: this doesn't just update count but also size bytes, maybe name it more properly (like IncrementCompletedBlobMetrics or something)